### DISCUSSION TITLE
Add a possibility for ImageUpdateAutomation to reference GitRepository from another namespace

### DISCUSSION THREAD

**Discussion by ilya-git** [fluxcd/flux2#1105] [Proposals]
Please see more discussions in this issue:
https://github.com/fluxcd/flux2/discussions/1017

The problem is that we would like to keep app-relevant stuff in it's own namespace, like e.g. team1, team2, etc., but since GitRepository itself is in the flux-system, everything else should be in the flux-system itself.

Maybe there is some other proper solution to this, but it would probably be a good idea to allow ImageUpdateAutomation to reference GitRepository from another namespace?

**Comment by stefanprodan** [comment]
Duplicate of: https://github.com/fluxcd/image-automation-controller/issues/85
